README.md

```diff
@@ -8,3 +8,9 @@ https://hacktoberfestnh.github.io
 - [Event release text (quoted)](docs/RELEASE-quoted.md)
 
 <small>Boostrat theme from https://bootstrapmade.com</small>
+
+## How to run
+
+You can use a simple HTTP server. Suggestions:
+- [🔗 Serve](https://www.npmjs.com/package/serve)
+- [🔗 HTTP-Server](https://www.npmjs.com/package/http-server)
```

Comment:
Don't need. Just open `index.html` in browser is enough.

But if you insist, please add python built-in http server, available in any linux distribution.

```
python2 -m SimpleHTTPServer 8000
```